app/models/miq_event_definition.rb

```diff
@@ -123,9 +123,8 @@ def self.seed
 
   def self.seed_default_events(event_defs)
     event_sets = MiqEventDefinitionSet.all.index_by(&:name)
-    fname = File.join(FIXTURE_DIR, "#{to_s.pluralize.underscore}.csv")
-    CSV.foreach(fname, :headers => true, :skip_lines => /^#/, :skip_blanks => true) do |csv_row|
-      event = csv_row.to_hash
+    fname = File.join(FIXTURE_DIR, "miq_event_definitions_sets.yml")
```

Comment:
I think this should be
```suggestion
    fname = File.join(FIXTURE_DIR, "miq_event_definitions.yml")
```

Reply:
`miq_event_definitions.yml` was already taken so we agreed to name the new yml file `miq_event_definitions_sets` with `miq_event_definition_sets` being the name of the other one

Reply:
That is super confusing.

Since

- `MiqEventDefinition.seed_default_events` uses `miq_event_definitions.csv`
- `MiqEventDefinition.seed_default_definitions` uses `miq_event_definitions.yml`
- `MiqEventDefinitionSet.seed` uses `miq_event_definition_set.csv`

I recommend

- Convert and rename `miq_event_definitions.csv` to `miq_event_definition_events.yml` 
- Keep `miq_event_definitions.yml` as is
- Convert and rename `miq_event_definition_set.csv` to `miq_event_definition_set.yml`

Reply:
will do

Reply:
gah I typod one of the file names (updated above)